### PR TITLE
skaffold 1.13.0

### PR DESCRIPTION
**Pull request by dtrodrigues**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by dawidd6** [comment]
GOPATH can be dropped.

**Comment by dawidd6** [comment]
This `dirty` git tree state is probably caused by `buildpath/.brew_home`. I wouldn't worry about it that much and just change the test to do something else.

File: Formula/skaffold.rb [hunk 1]
```diff
@@ -17,25 +17,20 @@ class Skaffold < Formula
...
+    (zsh_completion/"_skaffold").write output
   end
 
   test do
-    output = shell_output("#{bin}/skaffold version --output {{.GitTreeState}}")
-    assert_match "clean", output
+    df = testpath/"Dockerfile"
+    df.write <<~EOS
+      FROM scratch
+    EOS
```
Comment (SeekingMeaning) [inline Formula/skaffold.rb:32]:
```suggestion
    (testpath/"Dockerfile").write "FROM scratch"
```

File: Formula/skaffold.rb [hunk 2]
```diff
@@ -17,25 +17,20 @@ class Skaffold < Formula
...
+    output = shell_output("#{bin}/skaffold init --analyze").chomp
+    assert_equal "{\"dockerfiles\":[\"Dockerfile\"]}", output
```
Comment (SeekingMeaning) [inline Formula/skaffold.rb:34]:
```suggestion
    assert_equal '{"dockerfiles":["Dockerfile"]}', output
```

**Review by SeekingMeaning — APPROVED** [review]

**Pull request closed without merging**